Projects/Features/AuthFeature/Sources/SignUpViewModel.swift

```diff
@@ -47,6 +52,15 @@ public class SignUpViewModel: ViewModel {
         UserDefaults.standard.set(name, forKey: "USER_NAME")
         
         let savedName = UserDefaults.standard.string(forKey: "USER_NAME")
-        output.isSaved.send(savedName == name)
+        
+        createChannelUsecase.execute(name: "Guest")
+            .sink { _ in
+            } receiveValue: { [weak self] channelEntity in
+                UserDefaults.standard.set(channelEntity.id, forKey: "CHANNEL_ID")
+                let savedID = UserDefaults.standard.string(forKey: "CHANNEL_ID")
+                
+                self?.output.isSaved.send(savedName == name && savedID != nil)
```

Comment:
```suggestion
                self?.output.isSaved.send(savedName == name && savedID == channelEntity.id)
```

nil 값 확인이 아닌 `channelEntity.id`와 비교하는 것도 좋을 것 같아요.

Reply:
그방법도 좋은 것 같습니다! 다음에 수정해서 반영해두겠습니다!